cpp/benchmarks/CMakeLists.txt

```
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+
+project(CUSPATIAL_BENCHS LANGUAGES C CXX CUDA)
```

```suggestion
project(CUSPATIAL_BENCHMARKS LANGUAGES C CXX CUDA)
```

(or at least spell BENCHES correctly. :) )

---

Consistency! :laughing: 

https://github.com/rapidsai/cudf/blob/1046eed3b8a59792fa6ed5f66a34cc5b792ef929/cpp/benchmarks/CMakeLists.txt#L3